exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/validate/IteratorValidatorBatchIterator.java

```diff
@@ -47,6 +47,8 @@
   /** For logging/debuggability only. */
   private static volatile int instanceCount;
 
+  private final boolean isRepeatable;
```

Comment:
Similar to the previous variable's javadoc comment, pls add one here and just link to the comment in `IteratorValidator`.

Reply:
Done